src/Numerics/Optimizers/ExhaustiveOptimizer/exhaustiveoptimizer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#=========================================================================
+#
+#  Copyright NumFOCUS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+
+# Python example demonstrating itk.ExhaustiveOptimizer usage
+import sys
+from math import pi
+
+import itk
+
+EXIT_SUCCESS = 0
+EXIT_FAILURE = 1
+
+def main(argv):
+    if(len(argv) < 3):
+        raise Exception(f'Usage: {argv[0]} fixedImage movingImage')
+
+    dimension = 2
```

Comment:
Start with:

```
fixed_image = itk.imread(argv[1], itk.F
```

Note -- use snake_case per idiomatic Python for the variable names.

`dimension`, if needed, can be retrieved with `dimension = fixed_image.GetImageDimension()`.

Reply:
Some classes such as `Euler2DTransform` are only wrapped for or compatible with 2D images. How should it be expressed that the image dimension is fixed in this example, i.e. only 2D image input is allowed?

Snake case will be updated.

Reply:
The transform type can be conditional on the dimension like:

```
if dimension == 2:
    transform = itk.Euler2DTransform[...]
elif dimension == 3:
   transform = itk.Euler3DTransform[...]
else:
    raise RuntimeError(f'Unsupported dimension: ${dimension}')
```

Reply:
Resolved in most recent update.